docs/hello_nextflow/01_orientation.md

```diff
@@ -64,40 +59,20 @@ You should see the following output:
     ├── hello-world-8.nf
     ├── hello-world-9.nf
     ├── modules
-    │   └── local
-    │       ├── gatk
-    │       │   ├── haplotypecaller
-    │       │   │   └── main.nf
-    │       │   └── jointgenotyping
-    │       │       ├── main.nf
-    │       │       └── tests
-    │       │           └── inputs
-    │       │               ├── family_trio_map.tsv
-    │       │               ├── reads_father.bam.g.vcf
-    │       │               ├── reads_father.bam.g.vcf.idx
-    │       │               ├── reads_mother.bam.g.vcf
-    │       │               ├── reads_mother.bam.g.vcf.idx
-    │       │               ├── reads_son.bam.g.vcf
-    │       │               └── reads_son.bam.g.vcf.idx
-    │       └── samtools
-    │           └── index
-    │               └── main.nf
     └── nextflow.config
 
-12 directories, 43 files
+13 directories, 48 files
 
 ```
 
-Each file will be used in this training module.
-
-**The `data` directory** contains the input data we'll use in Part 2: Hello GATK, which uses an example from genomics to demonstrate how to build a simple analysis pipeline. The data is described in detail in that section of the training.
+**The `data` directory** contains the input data we'll use in Part 2: Hello Science, which uses an example from genomics to demonstrate how to build a simple analysis pipeline. The data is described in detail in that section of the course.
```

Comment:
```suggestion
**The `data` directory** contains the input data we'll use in Part 3: Hello Science, which uses an example from genomics to demonstrate how to build a simple analysis pipeline.
The data are described in detail in that section of the course.
```

Reply:
Oh no you're a "data are" person, and here I thought we were going to get along 🤣 (j/k)